### ISSUE TITLE
Fix TS compiler warnings

### ISSUE THREAD

**Issue by adidahiya** [palantir/tslint#547]
Some of our grunt tasks produce this warning:

```
WARNING: TypeScript does not allow external modules to be concatenated with --out. Any exported code may be truncated.  See TypeScript issue #1544 for more details.
Using tsc v1.6.0-dev.20150804
```

We should simplify our grunt builds and try to move away from concatenation via `--out`.

**Comment by adidahiya** [comment]
This will probably go away if we implement #465.
